### PR TITLE
New component - URL Preview

### LINKED ISSUE #15592: Create quo component URL Preview
# Feature Issue

Create new quo2 `URL Preview` component.

Details in Figma > Design System https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?node-id=19128-198770&t=fhiOBYdYdeD8xLhR-0

### Notes

This component will be used to display link previews in chats. Related to issue https://github.com/status-im/status-mobile/issues/15469

### PR DESCRIPTION
**Pull request by ilmotta**
Fixes https://github.com/status-im/status-mobile/issues/15592

### Summary

This PR implements the `URL Preview` component.

<table>
  <thead>
    <tr>
      <th>Light mode preview</th>
      <th>Dark mode preview</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td></td>
    </tr>
  </tbody>
</table>

### Review notes

- Please, if you have access to iOS, check that the component renders correctly.
- The red border around the icon has been reported in issue https://github.com/status-im/status-mobile/issues/15606
- This PR is the first in a series of PRs related to URL Previews.
- The loading state doesn't have a spinning icon on purpose. This will be done separately (issue yet to be created).

#### Platforms

- Android
- iOS

status: ready

### CODE REVIEW

**Review by smohamedjavid — APPROVED** [review]
LGTM 🚀 

Attaching screenshots for iOS.

<table>
  <thead>
    <tr>
      <th>Light mode preview</th>
      <th>Dark mode preview</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td></td>
    </tr>
  </tbody>
</table>

**Review by OmarBasem — APPROVED** [review]
Renders correctly on iOS 👍

**Review by ibrkhalil — APPROVED** [review]

File: src/quo2/components/links/url_preview/view.cljs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+     :number-of-lines     1
+     :style               (style/body)}
+    body]])
+
+(defn- clear-button
+  [{:keys [on-press]}]
+  [rn/touchable-opacity
+   {:on-press            on-press
+    :style               style/clear-button-container
+    :hit-slop            {:top 3 :right 3 :bottom 3 :left 3}
```
Comment (ulisesmac) [inline src/quo2/components/links/url_preview/view.cljs:39]: Very good :+1: 
Some components are small and this seems useful!

**Review by ulisesmac — APPROVED** [review]
Everything looks good :+1:

**Pull request merged**